<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dhx64Class</class>
 <widget class="QMainWindow" name="dhx64Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dhx64</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/dhx64/Resources/logo.png</normaloff>:/dhx64/Resources/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="labelLeft">
      <property name="text">
       <string>Left Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelRight">
      <property name="text">
       <string>Right Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReflashDeviceInfo"/>
   <addaction name="separator"/>
   <addaction name="actionLoadLeftImg"/>
   <addaction name="actionLoadRightImg"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionSaveImage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetDeviceManager">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>120</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Device Manager</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetCameraProperty">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>446</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Camera Property</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="3" column="0">
      <widget class="QCommandLinkButton" name="commandLinkButtonCatch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Catch Image</string>
       </property>
       <property name="icon">
        <iconset resource="scanner.qrc">
         <normaloff>:/scanner/Resources/Icons/catch.png</normaloff>:/scanner/Resources/Icons/catch.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCommandLinkButton" name="commandLinkButtonSaveLRImg">
       <property name="text">
        <string>Save LR Image</string>
       </property>
       <property name="icon">
        <iconset resource="scanner.qrc">
         <normaloff>:/scanner/Resources/Icons/save.png</normaloff>:/scanner/Resources/Icons/save.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxCamera">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Camera</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Capture Mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Software Trigger</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hardware Trigger</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderGain">
            <property name="maximum">
             <number>1023</number>
            </property>
            <property name="value">
             <number>400</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxGain">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>1023</number>
            </property>
            <property name="value">
             <number>400</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBoxSerialPort">
       <property name="title">
        <string>Serial Port</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCommandLinkButton" name="commandLinkButtonCheckport">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Check Serial Prot</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/dhx64/Resources/synchronized.png</normaloff>:/dhx64/Resources/synchronized.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCommandLinkButton" name="commandLinkButtonOpenClose">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Open Serial Port</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/dhx64/Resources/serialport.png</normaloff>:/dhx64/Resources/serialport.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Serial Port</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxSerialPort"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionReflashDeviceInfo">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/synchronized.png</normaloff>:/scanner/Resources/Icons/synchronized.png</iconset>
   </property>
   <property name="text">
    <string>reflashDeviceInfo</string>
   </property>
   <property name="toolTip">
    <string>Find Decices</string>
   </property>
   <property name="statusTip">
    <string>Find Decices</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/save.png</normaloff>:/scanner/Resources/Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>saveImage</string>
   </property>
   <property name="toolTip">
    <string>Save Image</string>
   </property>
   <property name="statusTip">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/connect.png</normaloff>:/scanner/Resources/Icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
   <property name="toolTip">
    <string>Connect Device</string>
   </property>
   <property name="statusTip">
    <string>Connect Device</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/dicconnect.png</normaloff>:/scanner/Resources/Icons/dicconnect.png</iconset>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disonnect Device</string>
   </property>
   <property name="statusTip">
    <string>Disonnect Device</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/start-here.png</normaloff>:/scanner/Resources/Icons/start-here.png</iconset>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/Stop1NormalBlue.png</normaloff>:/scanner/Resources/Icons/Stop1NormalBlue.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionLoadLeftImg">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/left.png</normaloff>:/scanner/Resources/Icons/left.png</iconset>
   </property>
   <property name="text">
    <string>LoadLeftImg</string>
   </property>
   <property name="toolTip">
    <string>Load Left Image</string>
   </property>
   <property name="whatsThis">
    <string>Load Left Image</string>
   </property>
  </action>
  <action name="actionLoadRightImg">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/right.png</normaloff>:/scanner/Resources/Icons/right.png</iconset>
   </property>
   <property name="text">
    <string>LoadRightImg</string>
   </property>
   <property name="toolTip">
    <string>Load Right Image</string>
   </property>
   <property name="whatsThis">
    <string>Load Right Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scanner.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderGain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxGain</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxGain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderGain</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
